### ISSUE TITLE
No response from SNMPD on port 161/udp

### ISSUE THREAD

**Issue by blfriedman** [mib1185/ha-addon-snmpd#17]
I have been unable to get SNMPD add-on working.  Install, configure and running on HAOS install on bare metal server.  The logfile shows no errors after printing "INFO: Starting the snmpd daemon..."

SNMPD 0.4.0
HAOS 14.1
Core 2024.12.5
Supervisor 2024.12.0
Frontend 20241127.8

When I use the protection off mode on the SSH add-on, running "docker ps -a" shows no incoming port mapping into the "mib85/amd64-ha-addon-snmpd:0.4.0" container.  I would expect to see an assignment such as 0.0.0.0:161->161/udp.

The configuration for the add-on does not include any options for the network setup, so I cannot add it.

**Comment by mib1185** [comment]
This add-on uses the host network mode, so you won't see any port mapping.

> I have been unable to get SNMPD add-on working.

Is this statement based only on the not shown port mapping, or do you really encounter an issue?

**Comment by blfriedman** [comment]
I am encountering an issue of timeout on the snmp request.

I normally use an snmp manager - the Mikrotik Dude server to query status - it is generating an error.

To verify, I ran this:

`brucef@officepc:~$ snmpwalk -Os -c public -v 1 10.10.1.126 .1.3.6.1.2.1`
`Timeout: No Response from 10.10.1.126`

based on your note, I checked netstat before and after starting the SNMPD add-on.
Before: 
`➜  ~ netstat -udp -nlp | grep 161`

After:
`➜  ~ netstat -udp -nlp | grep 161`
`udp        0      0 0.0.0.0:161             0.0.0.0:*                           -               `    

It would seem that something else is the problem, since a listener is created.  Do you have anything I can check to see why I am not getting a response?

**Comment by blfriedman** [comment]
I have continued to dig into this and found the problem is in the snmp command listed above.

If I change from version 1 to version 2c, the add-on works just fine.  Perhaps I had it set for v2c in the past and somehow it was changed to version 1 in my configuration of testing.

This works fine:
`brucef@officepc:~$ snmpwalk -Os -c public -v 2c 10.10.1.126 iso.3.6.1.2.1.1`
whereas this does not work
`brucef@officepc:~$ snmpwalk -Os -c public -v 1 10.10.1.126 iso.3.6.1.2.1.1`

I do not see a problem with the add-on and close this.  Thanks for your response
